src/components/Tx/TxHistory/TxEntry.jsx

```diff
@@ -0,0 +1,349 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import web3 from 'web3'
+import i18n from '../../../i18n'
+import Identicon from '../../Identicon'
+import { networkIdToName, hexToNumberString } from '../../../lib/util'
+
+const BN = web3.utils.BN
```

Comment:
1. Since you're just using the `fromWei` util function, I'd prefer we don't import a `web3` instance. Would recommend using our util function, `weiToEther`, when this PR is merged in: https://github.com/ethereum/ethereum-react-components/pull/76/files#diff-3f81a5326768d781840bf95be050c2c1R19

2. Similary, import and use BN functions from utils, so we can manage BN instances only in that one place.

Reply:
sounds good